libs/domain/picking/src/components/filters/filters.component.ts

```diff
@@ -90,14 +90,14 @@ export class FiltersComponent extends LitElement {
           <button>${i18n('picking.filter.reset')}</button>
         </oryx-button>
 
-        <form @submit=${this.onSubmit}>
+        <form @submit=${this.onSubmit} id="filter-form">
```

Comment:
Why do we need form id?

Reply:
To connect form with button which is outside of the form. To make it we need 'id' and 'form' attributes

Reply:
Can we use kepress listener on this button and don't pollute html with unnecessary id's?

Reply:
But the current approach is more HTML. It generates the additional `id` on the form but reduces using JS code.
@tobi-or-not-tobi what do you think?

Reply:
we do indeed not like IDs, even generated. Lets try an alternative solution, let me know if you need help.